composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=5.6",
+        "pear/pear-core-minimal": "^1.0",
```

Comment:
I think you'll need to remove the `minimal` entry in the `dev` section.

Reply:
Will do